Engine/src/Engine/UserInterface.cpp

```diff
@@ -0,0 +1,101 @@
+#include "UserInterface.h"
+
+ImGuiIO* UserInterface::s_IO = nullptr;
+ImGuiStyle* UserInterface::s_style = nullptr;
```

Comment:
s_ ? Is this a naming convention I wasn't aware of?

Reply:
I wasn't sure what the standard for static variables was. I assumed s_ since all member variables were m_.

Reply:
I'll add it to our standards then, it makes sense.